test/controllers/MemberDOBControllerSpec.scala

```diff
@@ -255,10 +256,11 @@ class MemberDOBControllerSpec extends UnitSpec with WithFakeApplication with I18
         )(any())
       }
 
-      "a request is made during the June 2018" in {
+      "a request is made which returns only a CY result (i.e. after 6th April)" in {
 
         when(mockRasConnector.getResidencyStatus(any())(any())).thenReturn(Future.successful(ResidencyStatus(SCOTTISH, None)))
         when(mockSessionService.cacheDob(Matchers.any())(Matchers.any())).thenReturn(Future.successful(Some(rasSession)))
```

Comment:
@nathan-maybrey When I removed this line it goosed the test completely - but it IS a duplicate of line 77 so I don't understand why its OK to leave it in!